app/assets/javascripts/game_moves.js

```diff
@@ -15,13 +24,13 @@ $(function() {
           piece_id: $(ui.draggable).data("piece_id")
         }
       })
-      .success(function(data){
-       $(ui.draggable).detach().css({top: 0,left: 0}).appendTo(this);
-         if (data.valid_move? === false) {
-         alert("Invalid move. Try again.");
-         window.location.reload();
-       }
-      });
+      // .success(function(data){
+      //    if (data.valid_move? === false) {
+      //    alert("Invalid move. Try again.");
+      //    window.location.reload();
+      //  }
+      // });
     }

```

Comment:
My games js file is slightly different- I was fiddling with it with the intention of getting a successful move to also update the turn value. As Zach said, it seems necessary to pull things together and make sure we have a coherent js file that's referencing the correct routes.